test/src/unit-filter-pipeline.cc

```diff
@@ -76,6 +76,9 @@ class Add1InPlace : public tiledb::sm::Filter {
   void dump(FILE* out) const override {
     (void)out;
   }
+  void dump_ss(std::stringstream& ss) const override {
+    (void)ss;
```

Comment:
If you simply omit the argument name `ss`, leaving the type, you don't need to dummy out a value to avoid an unused argument warning.